src/server/helpers/config.js

```diff
@@ -12,6 +12,7 @@ const CLIENT_SIDE_WHITELIST = [
   'NODE_ENV',
   'BASE_URL',
   'GITHUB_API_ENDPOINT',
+  'GITHUB_AUTH_CLIENT_ID',
```

Comment:
CLIEND_ID is not secret (it's part of app setting URL), so we can safely expose it to client.